src/robusta/core/discovery/discovery.py

```diff
@@ -83,6 +84,17 @@ def __create_service_info(
             is_helm_release=is_helm_release,
         )
 
+    @staticmethod
+    def create_service_info(obj: Union[Deployment, DaemonSet, StatefulSet, Pod, ReplicaSet]) -> ServiceInfo:
```

Comment:
any particular reason we need this wrapper method? cant we just use the exisiting `__create_service_info`?